excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/AbstractDisplayGroupParser.java

```diff
@@ -124,6 +125,9 @@ public AbstractDisplayGroupParser(ParseContext parseContext, ShowConditionParser
         group.setOrder(sample.getInteger(this.displayGroupOrder));
         group.setType(this.displayGroupType);
         group.setPurpose(this.displayGroupPurpose);
+        group.setWebhookMidEvent(parseWebhook(sample,
```

Comment:
`parseWebhook()` can return `null` - is `WebhookMidEvent` mandatory or optional?

Reply:
It is optional. It we set null it will end up in the response json like that:
    "wizard_pages": [
    {
      "id": "FR_solicitorCreate1",
      "label": null,
      "order": 1,
      "wizard_page_fields": [...],
      "show_condition": null,
      "callback_url_mid_event": null,
      "retries_timeout_mid_event": []
    },
    {
      "id": "FR_solicitorCreate2",
      "label": null,
      "order": 2,
      "wizard_page_fields": [...],
      "show_condition": null,
      "callback_url_mid_event": "http://test61.com",
      "retries_timeout_mid_event": [
        120
      ]
    }
  ],